### PR TITLE
Admin configuration

### PR DESCRIPTION
**Pull request by thomas-ayling**
Case study page now has edit functionality.
Title, cover image, overview, spotlight, body and links can all be edited and persisted.

### CODE REVIEW

File: src/components/case-study-single/EditBody.jsx [hunk 1]
```diff
@@ -2,42 +2,38 @@ import { React, useState, useEffect } from 'react';
 import Title from './Title';
 import MarkdownComponent from '../markdown-component/MarkdownComponent';
 import CaseStudyCarousel from '../carousels/case-studies-carousel/CaseStudyCarousel';
+import CaseStudyCarousel from '../carousels/case-studies-carousel/CaseStudyCarousel';
```
Comment (MihneaDinuGL) [inline src/components/case-study-single/EditBody.jsx:5]: Is this imported twice?

File: src/service/FeedbackService.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import axios from 'axios';
-const baseUrl = 'http://ec-acad-elb-a07a79316f54cbbf.elb.eu-west-2.amazonaws.com:3001/feedback';
-// const baseUrl = 'http://13.41.186.121:3001/feedback';
+// const baseUrl = 'http://ec-acad-elb-a07a79316f54cbbf.elb.eu-west-2.amazonaws.com:3001/feedback';
+const baseUrl = 'http://localhost:3001/feedback';
```
Comment (MihneaDinuGL) [inline src/service/FeedbackService.js:3]: Don't forget to change this once the backend is on the live server :)

**Review by MihneaDinuGL — APPROVED** [review]

File: src/components/case-study-single/EditBody.jsx (hunk 1 above)
Comment (thomas-ayling) [reply src/components/case-study-single/EditBody.jsx:5]: Resolved

File: src/service/FeedbackService.js (hunk 2 above)
Comment (thomas-ayling) [reply src/service/FeedbackService.js:3]: Thanks for the heads up

**Pull request merged**